src/lib389/lib389/cli_conf/replication.py

```diff
@@ -183,6 +183,13 @@ def enable_replication(inst, basedn, log, args):
         # rid is good add it to the props
         repl_properties['nsDS5ReplicaId'] = args.replica_id
 
+    # Validate consumer and hub settings
+    elif role == "consumer" or role == "hub":
+        # Check Replica ID
+        if args.replica_id is not None or args.replica_id != 65535:
+            #Error, Replica ID cannot be specified for consumer and hub roles
```

Comment:
Should have a space after the "#"

Reply:
Added a space and checked for other formatting thank you!